lib/benchee/formatters/unit.ex

```diff
@@ -0,0 +1,118 @@
+defmodule Benchee.Unit do
+  @moduledoc """
+  Provides scaling and labeling functions for numbers. Use "count" functions
+  for numbers that represent countable things, such as iterations per second.
+  Use "duration" functions for numbers that represent durations, such as run
+  times.
+  """
+
+  @type unit :: atom
+  @type scaled_number :: {number, unit}
+
+  # In 1.3, this could be declared as `keyword`, but use a custom type so it
+  # will also compile in 1.2
+  @type options ::[{atom, atom}]
+
+  @doc """
+  Scales a number in a domain's base unit to an equivalent value in the best
+  fit unit. Results are a `{number, unit}` tuple. See `Benchee.Unit.Count` and
+  `Benchee.Unit.Duration` for examples
+  """
+  @callback scale(number) :: scaled_number
+
+  @callback format(number) :: String.t
+
+  @callback best(list, options) :: unit
+
+  @callback label(unit) :: String.t
+
+  @callback magnitude(unit) :: number
+
+  def float_precision(float) when float < 0.01, do: 5
+  def float_precision(float) when float < 0.1, do: 4
+  def float_precision(float) when float < 0.2, do: 3
+  def float_precision(_float), do: 2
```

Comment:
Not really...it's imported by the console formatter for now, until I actually use the units code in the formatter, at which point it won't be necessary. But yes, it should go to Common, good catch!